asyncpgsa/record.py

```diff
@@ -1,23 +1,26 @@
-
+from sqlalchemy import Column
 
 class Record:
     __slots__ = ('row',)
 
-    def __str__(self):
-        return str(self.row)
-
     def __init__(self, row):
         self.row = row
 
     def __getattr__(self, item):
         try:
             return self.row[item]
         except KeyError:
-            raise AttributeError("'Row' object has no attribute '{}'"
-                                 .format(item))
+            try:
+                return self.row.__getattribute__(item)
```

Comment:
Can you change this to be `return getattr(self.row, item)`? (getattr does MORE than just call __getattribute__)
Unless of course you have a reason you think it should use the dunder directly.